integration/shareddata/scalars.go

```diff
@@ -24,36 +24,37 @@ import (
 )
 
 const (
-	doubleMaxPrec = float64(1<<53 - 1) // 9007199254740991.0:    largest double values that could be represented as integer exactly
-	doubleBig     = float64(1 << 61)   // 2305843009213693952.0: some number larger than safe integer (doubleBig+1 == doubleBig)
-	longBig       = int64(1 << 61)     // 2305843009213693952:   same as doubleBig but integer
+	doubleMaxPrec float64 = float64(1<<53 - 1) // 9007199254740991.0:    largest double values that could be represented as integer exactly
+	doubleBig     float64 = float64(1 << 61)   // 2305843009213693952.0: some number larger than safe integer (doubleBig+1 == doubleBig)
+	longBig       int64   = int64(1 << 61)     // 2305843009213693952:   same as doubleBig but integer
```

Comment:
Can it be `int64(doubleBig)` then if it is the same? And then we probably test in the `init` function that it is actually the same

Reply:
The second part wasn't done

Reply:
It was already there: https://github.com/FerretDB/FerretDB/pull/3024/files/ac63e6ed60f90c0d87759bead40b480b0b9ee7fb#diff-8307cfe62efa2f64d4ef2d0aff407a8af0986af9f9efe7f98de217e13fe89297R348